### PR TITLE
[#151266508] Send Rogue campaign signup events to Gambit Conversations

### PR DESCRIPTION
**Pull request by sergii-tkachenko**
#### What's this PR do?
- Creates `gambit-campaign-signup-relay` queue which recieves campiang signup messages
- Creates `gambit-campaign-signup-relay` worker which listenes for new messages in `gambit-campaign-signup-relay` queue and relayes them to Gambit Conversation in [specified](https://github.com/DoSomething/gambit-conversations/blob/master/documentation/endpoints/send-message.md) format

#### How should this be manually tested?
- Sign up for a Thor campaign which is enabled in Gambit Campaigns Staging

#### What are the relevant tickets?
Pivotal [151266508](https://www.pivotaltracker.com/story/show/151266508)
Ref https://github.com/DoSomething/gambit-conversations/pull/155

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ gambit-chatbot-mdata-proxy: npm run worker gambit-chatbot-mdata-proxy
 customer-io-update-customer: npm run worker customer-io-update-customer
 customer-io-campaign-signup: npm run worker customer-io-campaign-signup
 customer-io-campaign-signup-post: npm run worker customer-io-campaign-signup-post
+gambit-campaign-signup-relay: npm run worker gambit-campaign-signup-relay
```
Comment (rapala61) [inline Procfile]: A suggestion, naming here could match better the term we have been using for these messages, like `web-campaign-signup-gambit-relay`.

**Comment by sergii-tkachenko** [comment]
Staging tests are blocked by Rogue Thor maintenance.

File: src/workers/GambitCampaignSignupRelayWorker.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+  async consume(message) {
+    const data = message.getData();
+
+    // See https://github.com/DoSomething/gambit-conversations/blob/master/documentation/endpoints/send-message.md
+    const body = {
+      northstarId: data.northstar_id,
+      campaignId: data.campaign_id,
+      // Web signups are consdered external for Gambit.
+      template: 'externalSignupMenuMessage',
```
Comment (rapala61) [inline src/workers/GambitCampaignSignupRelayWorker.js]: TL;DR: Change this to `externalSignupMenu` please!

This is correct based on the [supported templates](https://github.com/DoSomething/gambit-conversations/blob/master/documentation/endpoints/send-message.md#campaign) section in the docs, but is actually not accurate (Apologies in behalf of the Gambit kingdom). I'll be updating this in the docs ASAP.

**Comment by rapala61** [comment]
Misleading `send-message` route issue: https://github.com/DoSomething/gambit-conversations/issues/159

File: Procfile (hunk 1 above)
Comment (sergii-tkachenko) [reply Procfile]: I would argue that "campaign signup" is now not a web campaign signup. These, in fact, are coming from Rogue and could be as different as rep phone calls or offline actions.

File: test/workers/GambitCampaignSignupRelayWorker.test.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+  gambitWorker.checkRetrySuppress(retrySuppressResponse).should.be.true;
+
+
+  // Normal Gambit 422 response
+  const normalFailedResponse = new Response(
+    'Unknown Gambit error',
+    {
+      status: 422,
+      statusText: 'Unknown Gambit error',
```
Comment (rapala61) [inline test/workers/GambitCampaignSignupRelayWorker.test.js]: I just wanted to say, this test should pass! 🌮

File: src/workers/GambitCampaignSignupRelayWorker.js (hunk 2 above)
Comment (sergii-tkachenko) [reply src/workers/GambitCampaignSignupRelayWorker.js]: NP, fixing!

**Comment by sergii-tkachenko** [comment]
Testing is still blocked by broken Ashes -> Rogue integration. Getting "There was an error processing your request" error.

**Comment by sergii-tkachenko** [comment]
Discovered undesired retry loop for users with no phone number:

```
at=warning application=blink env=staging code=error_gambit_campaign_signup_relay_response_not_200_retry worker=GambitCampaignSignupRelayWorker request_id=dc052117-3a70-426f-a57f-0b5d0b8220be response_status=500 response_status_text="Internal Server Error" {"message":"The string supplied did not seem to be a phone number"}
```

**Comment by sergii-tkachenko** [comment]
Ref https://github.com/DoSomething/blink/pull/147

**Comment by sergii-tkachenko** [comment]
Ref https://github.com/DoSomething/gambit-conversations/pull/166

**Pull request merged**